### PR TITLE
Deploy kubernetes control plane

### PR DESCRIPTION
**Pull request by ueokande**
Deploy the following components to control plane nodes.
- [rivers](https://github.com/cybozu-go/cke-tools/tree/master/cmd/rivers) (TCP reverse proxy)
- kube-apiserver
- kube-controller-manager
- kube-scheduler

### CODE REVIEW

**Comment by dulltz** [comment]
If you want to execute only specific test cases of mtest Please execute mtest as follows:

```console
$ cd mtest
$ make placemat
```

then another console

```console
$ cd mtest
$ make test-light TARGET="should deploy HA control plane"
```

File: mtest/kubernetes_strategy_test.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+			return checkKubernetesClusterStatus(status, controlPlanes, workers)
+		}).Should(BeTrue())
+
+		By("Killing the active service")
+		leader := make(map[string]string)
+		for _, service := range []string{"kube-controller-manager", "kube-scheduler"} {
+			stdout, _, err := execAt(node1, getLeaderCommands(service)...)
+			Ω(err).ShouldNot(HaveOccurred())
+			Ω(len(stdout)).ShouldNot(BeZero())
```
Comment (ueokande) [inline mtest/kubernetes_strategy_test.go]: Use `Expect`

File: mtest/kubernetes_strategy_test.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+			holderIdentity := string(stdout)
+			fmt.Printf("current active %s is %s\n", service, holderIdentity)
+
+			leader[service] = strings.SplitN(string(stdout), "_", 2)[0]
+
+			stdout, _, err = execAt(os.Getenv(strings.ToUpper(leader[service])), "docker", "kill", service)
+			Ω(err).ShouldNot(HaveOccurred())
```
Comment (ueokande) [inline mtest/kubernetes_strategy_test.go]: `Expect`

File: mtest/kubernetes_strategy_test.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+				if string(stdout) == leader[service] {
+					fmt.Printf("active %s has not switched yet\n", service)
+					return false
+				}
+				return true
+			}).Should(BeTrue())
+		}
+
+		By("Checking component statuses are healthy")
+		Ω(checkComponentStatuses(node1)).Should(BeTrue())
```
Comment (ueokande) [inline mtest/kubernetes_strategy_test.go]: `Expect`

**Comment by ueokande** [comment]
lgtm

**Pull request merged**